### PR TITLE
Audience manager

### PR DESCRIPTION
**Pull request by Ciri-Zireael**
# Audience Manager
this component is responsible for placing avatars in the **random** seats. 

## The requirements: 
- models of chairs in the scene tagged as "chair"

## Input:
- list of Avatar prefabs
- number of seats we want to fill  
  
<br> 
The script is currently attached to an empty object called <em>AudienceManager</em> (very original) but is not yet made into a prefab (cause I forgot to do it xd (but that can be easily changed))

### CODE REVIEW

**Comment by Ciri-Zireael** [comment]
I forgot that I made a prefab for the seat position that I ended up never using 💀 (will need to delete later to avoid clutter)

File: Assets/Scripts/AudienceManager.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+public class AudienceManager : MonoBehaviour
+{
+
+	[SerializeField] private GameObject[] avatars;
```
Comment (TerMyY) [inline Assets/Scripts/AudienceManager.cs:10]: I'm assuming this extra tab wasn't intentional

File: Assets/Scripts/AudienceManager.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        while (selectedSeats.Count < numberOfSeats)
+        {   
+            int randomIndex = Random.Range(0, _allSittingPositions.Count);
+            selectedSeats.Add(_allSittingPositions[randomIndex]);
+        }
+
+        return selectedSeats;
+    }
+
+    void Update()
```
Comment (TerMyY) [inline Assets/Scripts/AudienceManager.cs:73]: I think it's better to remove empty methods, just for the sake of code looking more clean and pretty

**Review by TerMyY — APPROVED** [review]
The code is good for our current needs and overall I haven't had any issues testing the system. Great job!

**Pull request merged**